demo/app-root.ts

```diff
@@ -46,6 +57,26 @@ export class AppRoot extends LitElement {
   @query('#include-selected-option')
   private includeSelectedOptionCheck!: HTMLInputElement;
 
+  constructor() {
+    super();
+    // Copy sample userlist data
+    this.userlistData = userlistTestData.map(item => item);
+
+    // Initialize selected count for main button icon state
+    this.selectedCount = this.userlistData.filter(
+      item => item.item_is_member
+    ).length;
+
+    // Listen for closeDropdown event from item-userlists
+    const eventListener = (e: CustomEvent) => {
+      this.selectedCount = e.detail.selected as number;
+      // Set selected count for main button icon state
+      console.log('Selected count: ', e.detail.selected);
+    };
+    // eslint-disable-next-line no-undef
+    this.addEventListener('selectDropdown', eventListener as EventListener);
```

Comment:
Q: Does TypeScript complain without the type cast?

Reply:
Yes it does. I couldn't find out why in several tries.
I searched "EventListener" in Collection Browser code and found this.

I've bashed at TypeScript listeners custom events type safety several times, got it to work, then Tuesday(?) lost the code in a mistaken `git restore`. Didn't remember how I did it, and couldn't seem to replicate it.